singer_sdk/streams/rest.py

```diff
@@ -140,7 +140,7 @@ def requests_session(self) -> requests.Session:
             The `requests.Session`_ object for HTTP requests.
 
         .. _requests.Session:
-            https://requests.readthedocs.io/en/latest/api/#request-sessions
+            https://requests.readthedocs.io/en/latest/api.html?#requests.Session
```

Comment:
Is the `?` needed?

Reply:
Appears not, it must have come out of the referral from the search page on the site - I can remove it